### DISCUSSION TITLE
Backport WUA 7.4.7600.256 to Windows 2000

### DISCUSSION THREAD

**Discussion by XPMilkChan** [LegacyUpdate/LegacyUpdate#35] [Ideas]
I noticed that your Windows Update
If running on Windows 2000 with the latest windows update agent 7.6 of Windows 2000 7.4.7600.226 it will not work.
I checked WindowsUpdateAgent-7.6-x86(256).exe Saw that it was blocked for Windows 2000, I modified the PE part to make the program run on 2000, can you please put it in your web page?
I think it's necessary
![bandicam 2022-11-13 19-48-43-545](https://user-images.githubusercontent.com/106049626/201522520-ae1c168b-d1ac-4781-b567-2a29cf5ba272.jpg)
![image](https://user-images.githubusercontent.com/106049626/201522027-fc5f6a35-e4d8-4d09-adb6-bf75bb48caa7.png)
![bandicam 2022-11-13 16-45-58-067](https://user-images.githubusercontent.com/106049626/201522038-a16b79d8-9390-46e2-bb48-b399d7d04b79.jpg)
I noticed that there is something wrong with your icon.
![bandicam 2022-11-13 17-05-53-734](https://user-images.githubusercontent.com/106049626/201522097-5a75ffc6-04df-4da4-a562-80200cbc10e4.jpg)

DL : https://mega.nz/file/CoAmABga#vnwsaNXEXopM5CUImD8owRS0Wdt0XE0Fl3n6GmVCgXE

WUA-Downlevel.exe for nt5x 
Note The first time may fail.
Had to reset again for this to work.

**Comment by kirb** [comment]
This is really cool, good work and thanks for sharing! I’d want to have it binary edited on the user’s machine directly by the installer just to provide some assurance that it’s all legit Microsoft binaries, and also to hopefully minimise antivirus false positives. I hope you don’t mind if I diff your binaries against the originals to do that?

**Comment by XPMilkChan** [comment]
> This is really cool, good work and thanks for sharing! I’d want to have it binary edited on the user’s machine directly by the installer just to provide some assurance that it’s all legit Microsoft binaries, and also to hopefully minimise antivirus false positives. I hope you don’t mind if I diff your binaries against the originals to do that?

Yes, you can extract my binary.
I downloaded from your web page.

**Comment by makuhlmann** [comment]
> the latest windows update agent 7.6 of Windows 2000 7.4.7600.226 it will not work

The latest that can be installed without any tricks is 7.4.7600.243.

> I think it's necessary

I don't see why force installing 7.4.7600.256 would help anyone. The older versions are perfectly capable of updating the system without any installation hacks.

**Comment by XPMilkChan** [comment]
> > the latest windows update agent 7.6 of Windows 2000 7.4.7600.226 it will not work
> 
> The latest that can be installed without any tricks is 7.4.7600.243.
> 
> > I think it's necessary
> 
> I don't see why force installing 7.4.7600.256 would help anyone. The older versions are perfectly capable of updating the system without any installation hacks.

thank you ok I see the significance, I checked one of the creators UpdatePack and it works on WIndows 2000.
![bandicam 2022-11-15 18-15-56-591](https://user-images.githubusercontent.com/106049626/201906447-8367a124-7dea-432a-8c4c-6afa4f3912ed.jpg)
https://ryanvm.net/forum/viewtopic.php?t=5326

**Comment by kirb** [comment]
I wouldn’t mind backporting 256, because that version brings some security improvements (it was released in response to [Flame](https://en.wikipedia.org/wiki/Flame_\(malware\))). I’d want to provide both patched 256 and unpatched 243 as options, because I realise some may prefer a build that’s original over one that’s technically more secure, but isn’t “supposed” to work on 2000.
